spec/defines/requirements_spec.rb

```diff
@@ -53,7 +53,7 @@
         it { is_expected.to contain_package('gunicorn') }
         it { is_expected.to contain_file('/requirements.txt').with_owner('root').with_group('root') }
 
-        if facts[:os]['name'] == 'Gentoo'
+        if ['FreeBSD', 'Gentoo'].include?(facts[:os]['name'])
```

Comment:
rubocop doesn't like that. Our `rubocop:auto_correct` rake task might fix that automatically for you

Reply:
Ah, fixed…